src/datepicker/__tests__/stateful-calendar.scenario.js

```diff
@@ -13,5 +13,15 @@ import {StatefulCalendar} from '../index.js';
 export const name = 'stateful-calendar';
 
 export const component = () => (
-  <StatefulCalendar highlightedDate={new Date('2019-02-14T10:00:00Z')} />
+  <>
+    <StatefulCalendar highlightedDate={new Date('2019-02-14T10:00:00Z')} />
+    <br />
+    <StatefulCalendar
```

Comment:
Based on the disabled logic change and the min/max date provided here, I would expect the left/right arrow buttons in these new VRTs to be disabled as well. They look enabled in the VRTs.

Reply:
why is that? the `maxDate` is in 2001, the `minDate` is in 1999, and the initial value is in 2000 - so imho, the arrows should be enabled. Am I missing something?

On the master branch, the bug was that you couldn't paginate to 2000, because the prop was not taken into account. This was added to make sure it's enabled.

Reply:
Oh sorry, I overlooked the years and only compared the month/day values